src/Cooked/MockChain/BlockChain.hs

```diff
@@ -381,6 +382,33 @@ txSkelInputDataAsHashes skel = do
   (Map.elems -> inputTxOuts) <- txSkelInputUtxos skel
   catMaybes <$> mapM outputToDatumHashM inputTxOuts
 
+-- | This creates a payment from an existing TxOut. This is not trivial because
+-- we need to reconstruct a bunch of information using data withing the blockchain
```

Comment:
There's a typo, but since we're at it, I would rather use `from`

```suggestion
-- we need to reconstruct a bunch of information using data from the blockchain
```